src/app.ts

```diff
@@ -0,0 +1,33 @@
+import express from 'express';
+
+import ChatterService from './services/ChatterService';
+
+const app = express();
+app.use(express.json());
+
+let viewerwersPoint = new Map();
+
+setInterval(async () => {
+  try {
+    const viewers = await ChatterService.getViewers();
+
+    viewers.forEach((viewer) => {
+      let point = viewerwersPoint.get(viewer) ?? 0;
+      viewerwersPoint.set(viewer, ++point);
```

Comment:
Sobre o `++`
O ESLint tem regras e maior parte dos styleguides usam ela, pra não utilizar esse operador. Não estou certo da motivação. Achei na internet que Douglas Crockford no seu livro Javascript: The Good Parts, ele diz:

> ++ and --
The ++ (increment) and -- (decrement) operators have been known to contribute to bad code by encouraging excessive trickiness. They are second only to faulty architecture in enabling to viruses and other security menaces. There is a plusplus option [in JSLint] that prohibits the use of these operators.

Eu não sei exatamente se é uma boa opção. Isso seria uma longa discussão, mas de qualquer forma é claramente algo que pode nos confundir, considerando que `++point` é bem diferente de `point++`.

>

Reply:
Eu concordo. Existe uma Issue aberta com relação ao uso de uma database para armazenar os viewers e seus pontos e acredito que isso será a solução para essa implementação.